edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -861,6 +894,47 @@ VolumeOptions GetVolumesFromModule(IModule<AgentDocker.CombinedDockerConfig> mod
                 : Option.None<(List<V1Volume>, List<V1VolumeMount>, List<V1VolumeMount>)>();
         }
 
+        private async Task GetOrCreatePersistentVolumeClaim(string name, bool readOnly)
+        {
+            var listResult = await this.client.ListNamespacedPersistentVolumeClaimAsync(KubeUtils.K8sNamespace);
+
+            var foundPvc = listResult.Items.SingleOrDefault(item => !string.IsNullOrWhiteSpace(item.Metadata.Name) && item.Metadata.Name == name);
```

Comment:
Is the `IsNullOrWhiteSpace` check really needed? Wouldn't the `item.Metadata.Name == name` check handle that?

Reply:
Or maybe make it: `item?.Metadata?.Name == name`.